src/AffineTraversal.js

```diff
@@ -10,7 +10,7 @@ class AffineTraversalT {
 
   over = (f, obj) => {
     const v = this.preview(obj)
-    return v === null ? obj : this.set(f(v), obj)
+    return v === undefined ? obj : this.set(f(v), obj)
```

Comment:
if you cast to a falsy value `!v`, you will check for both undefined and null! https://developer.mozilla.org/es/docs/Glossary/Falsy

Reply:
As mentioned above, checking falsy won't work because 0, `false`, empty string, all those are also falsy.